app/models/product.rb

```diff
@@ -1,8 +1,30 @@
 class Product < ApplicationRecord
-  validates :title, :description, :image_url, presence: { message: "can't be blank"}
+  class PriceValidator < ActiveModel::Validator
```

Comment:
let's move these validators to new files called price_validator and url_validator etc.